test-suite/shared/src/test/require-jdk7/org/scalajs/testsuite/javalib/lang/CharacterTestOnJDK7.scala

```diff
@@ -332,4 +332,32 @@ class CharacterTestOnJDK7 {
     assertFalse(Character.isIdeographic(958559))
     assertFalse(Character.isIdeographic(999076))
   }
+
+  @Test
+  def shouldProvideIsSurrogate(): Unit = {
+    //non-surrogate
+    assertFalse(Character.isSurrogate((Character.MIN_SURROGATE - 1).toChar))
+    assertFalse(Character.isSurrogate((Character.MAX_SURROGATE + 1).toChar))
+    assertFalse(Character.isSurrogate('a'))
+    assertFalse(Character.isSurrogate('7'))
+    assertFalse(Character.isSurrogate('ö'))
+    assertFalse(Character.isSurrogate('\t'))
+
+    //high surrogates
+    assertTrue(Character.isSurrogate(Character.MIN_SURROGATE).toChar)
```

Comment:
[The CI says](https://scala-webapps.epfl.ch/sjsjenkins/blue/organizations/jenkins/scalajs-main/detail/PR-3316/4/pipeline/52):
```
[error] .../test-suite/shared/src/test/require-jdk7/org/scalajs/testsuite/javalib/lang/CharacterTestOnJDK7.scala:347: value toChar is not a member of Boolean
[error]     assertTrue(Character.isSurrogate(Character.MIN_SURROGATE).toChar)
[error]                                                               ^
[error] .../test-suite/shared/src/test/require-jdk7/org/scalajs/testsuite/javalib/lang/CharacterTestOnJDK7.scala:355: value toChar is not a member of Boolean
[error]     assertTrue(Character.isSurrogate(Character.MAX_SURROGATE).toChar)
[error]                                                               ^
[error] two errors found
[error] (testSuite/test:compileIncremental) Compilation failed
```
The `.toChar` should not be there. This applies to this line, and to the one down below under `low surrogates`. The 2 lines in `non-surrogate` are correct.

Reply:
Ugh, sorry for all the bother @sjrd - this would have been much less work for you if you just did it yourself (or if I properly payed attention). Will fix (properly).